manual/dmc.tex

```diff
@@ -99,8 +99,8 @@ \section{Diffusion Monte Carlo}
 \item \texttt{substeps}.  Same as BranchInterval.
 
 
-\item \texttt{nonlocalmoves}. Evaluate pseudopotentials using one of the nonlocal move algorithms such as t-moves.  Setting nonlocalmoves to `no' will impose the locality approximation. `yes/v0' implements the algorithm in the 2006 Casula paper~\cite{Casula2006} and `v1' implements the v1 algorithm in the 2010 Casula paper~\cite{Casula2010}.
-The v1 algorithm is size-consistent and an important advance over the previous v0 non-size-consistent algorithm. Investigating the importance of size consistency is highly recommended.
+\item \texttt{nonlocalmoves}. Evaluate pseudopotentials using one of the nonlocal move algorithms such as t-moves.  Setting nonlocalmoves to `no' will impose the locality approximation. `yes/v0' implements the algorithm in the 2006 Casula paper~\cite{Casula2006} and `v1' implements the v1 algorithm in the 2010 Casula paper~\cite{Casula2010} and `v3' implements an algorithm similar to v1 but much faster.
+The v1 and v3 algorithms are size-consistent and an important advance over the previous v0 non-size-consistent algorithm. Investigating the importance of size consistency is highly recommended.
```

Comment:
Update the language here to be more precise and give more details: is v3 exactly the same in terms of answer as v1 or is it an approximation? Why is it faster?